rewrite-test/src/main/kotlin/org/openrewrite/java/JavaTemplateTest.kt

```diff
@@ -389,6 +389,90 @@ interface JavaTemplateTest : JavaRecipeTest {
         """
     )
 
+    @Test
+    fun replaceFieldAccess(jp: JavaParser) = assertChanged(
```

Comment:
As written exampleField1 won't be type-attributed. 
You should fix the visitor/test so that's the case and add an `afterCondition` ensuring that it is